src/features/adr/pages/hooks/useSubmitADR.tsx

```diff
@@ -4,9 +4,10 @@ import { getUrl } from "../../../../shared/utils";
 import { useADRSchemaQuery } from "./useADRSchemaQuery";
 
 export const submitADR = (formSchemaID: string) => (adr: any) => {
+  const { patient } = adr;
   return axios.post(
     getUrl("adverse_drug_reaction"),
-    { data: adr, form_schema_ID: formSchemaID },
+    { data: adr, form_schema_ID: formSchemaID, name_ID: patient?.ID },
```

Comment:
set the patient ID so that the linked record in the ADR saves correctly in desktop.